CHANGELOG.md

```diff
@@ -38,6 +38,8 @@ per cluster.
 - Rename NodeType and tags from Master to HeadNode.
 - Remove Ganglia support.
 - Add support for associating an existing Elastic IP to the head node.
+- Encrypt EBS volumes (both the root EBS volumes and shared EBS volumes) by default. 
+  Note that if the scheduler is AWS Batch, the root volumes of the compute nodes cannot be encrypted by parallelcluster.
```

Comment:
nit: Encrypt root EBS volumes and shared EBS volumes by default
nit: ParallelCluster

Reply:
Done